STX.REST.RESTFulSense.Clients/Brokers/Https/IHttpBroker.cs

```diff
@@ -2,8 +2,17 @@
 // Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
 // ----------------------------------------------------------------------------------
 
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+
 namespace STX.REST.RESTFulSense.Clients.Brokers.Https
 {
-    internal partial interface IHttpBroker 
-    { }
+    internal partial interface IHttpBroker
```

Comment:
Interface could be without "partial", same as @glhays suggested on Discord for the class.

Reply:
Resolved